docs/content/python/getting-started/quickstart/index.md

```diff
@@ -6,124 +6,120 @@ description: |
 weight: 1
 ---
 
-<!--
+# Quickstart
```

Comment:
There's not too much context on what Oso is in this guide. Just confirming, our assumption is someone will have already read a bit before coming here?

Reply:
Yeah in general I think it's good not to spread contextual info all over the place. If you're confused what Oso is, this probably isn't the guide you want.